scripts/e2e-test.sh

```diff
@@ -3,18 +3,65 @@
 set -eux
 
 SCRIPTDIR=$(dirname "$(realpath "$0")")
-REGISTRY=$1
 TESTS="install basic_volume_io"
+DEVICE=
+REGISTRY=
+TAG=
 
-# TODO: Add proper argument parser
-if [ -z "$REGISTRY" ]; then
-  echo "Missing parameter registry"
+help() {
+  cat <<EOF
+Usage: $(basename $0) [OPTIONS]
+
+Options:
+  --device <path>           Device path to use for storage pools.
+  --registry <host[:port]>  Registry to pull the mayastor images from.
+  --tag <name>              Docker image tag of mayastor images (default "ci")
+
+Examples:
+  $(basename $0) --registry 127.0.0.1:5000 --tag a80ce0c
```

Comment:
FYI I tend to use "$0" as command then can be copy-pasted and will work. YMMW.

Nice to have help btw :+1:

Reply:
agree. this was taken from another script of ours.